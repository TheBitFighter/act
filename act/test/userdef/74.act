template<pint self>
defproc test(bool foo)
{
  bool bar;
}
