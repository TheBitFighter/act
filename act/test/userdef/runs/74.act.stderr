ERROR: File `74.act', line 1, col 14
	Parameter name cannot be the reserved identifier `self'.
